chutney/action-impl/src/main/java/com/chutneytesting/action/kafka/KafkaBasicConsumeAction.java

```diff
@@ -204,18 +208,22 @@ private void addMessageToResultAndCountDown(Map<String, Object> message) {
         countDownLatch.countDown();
     }
 
-    private Object extractPayload(ConsumerRecord<String, String> record) {
+    private Object extractPayload(ConsumerRecord<?, ?> record) {
         if (recordContentType.getSubtype().contains(APPLICATION_JSON.getSubtype())) {
-            try {
-                return OBJECT_MAPPER.readValue(record.value(), Map.class);
-            } catch (IOException e) {
-                logger.info("Received a message, however cannot read it as Json fallback as String.");
+            if (record.value() instanceof String) {
+                try {
+                    return OBJECT_MAPPER.readValue((String) record.value(), Map.class);
```

Comment:
you can use this sugar syntax to avoid cast : 
```java
if (value instanceof String stringValue) {
            try {
                return OBJECT_MAPPER.readValue(stringValue, Map.class);